wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -457,5 +459,8 @@ api = toSwagger walletAPI
     , defaultPerPage = fromString (show defaultPerPageEntries)
     , accountExample = toS $ encodePretty (genExample @[Account])
     , accountExtendedExample = toS $ encodePretty (genExample @(ExtendedResponse [Account]))
+    , walletErrorList = T.intercalate ", " $ map (wrapWith "`") Errors.allErrorsList
```

Comment:
This is a good start, which we can extend in the future with our `Generic` derivation to also get the error codes and a simple example rendering as a JSON value. For this first iteration is more than enough.

I propose to rename `wrapWith` with `surroundedBy`, but I guess it's just personal preference 😉

Reply:
+1. Will update